src/super_gradients/common/sg_loggers/deci_platform_sg_logger.py

```diff
@@ -26,17 +26,9 @@ def __init__(self, **kwargs):
         if _imported_deci_lab_failure is not None:
             raise _imported_deci_lab_failure
 
-        auth_token = os.getenv("DECI_PLATFORM_TOKEN")
-        if auth_token is None:
-            raise ValueError(
-                'The environment variable "DECI_PLATFORM_TOKEN" is required in order to use '
-                "DeciPlatformSGLogger. Please set it with your own credentials "
-                "(available in https://console.deci.ai/settings)"
-            )
-
         super().__init__(**kwargs)
         self.platform_client = DeciPlatformClient()
-        self.platform_client.login(token=auth_token)
+        self.platform_client.login(token=os.getenv("DECI_PLATFORM_TOKEN"))
```

Comment:
but what if DECI_PLATFORM_TOKEN is not set?

Reply:
If DECI_PLATFORM_TOKEN is not set, then the client will ignore directly look for a token stored locally.

Basically, the question is do we want to check env variable or local token first?  My motivation for checking env variable first are:
- This allows us to connect with a different token easily, by just setting the env variable. In most situations it is not useful, but I see a few edge cases where this can help. Otherwise, you need to delete the local token manually first.
- The client code is coded in a way that currently only supports this flow. The workaround would be to use private methods to get the local token, or to ask to the platform team to implement this env variable check within the client
- I don't see a downside of doing so, but maybe I am not thinking of something?

What's your thought?